ui/server/src/main/scala/pl/touk/nussknacker/ui/api/ManagementResources.scala

```diff
@@ -100,6 +103,26 @@ object ManagementResources {
 
 }
 
+class DeploymentComment(private val comment: String) {
+  def value: String = comment
+}
+
+object DeploymentComment {
+  def apply(comment: String, settings: DeploySettings): Either[CommentNotMatchingValidationPattern, DeploymentComment] = {
```

Comment:
Maybe Validated from cats instead of Either ?

Reply:
you're right it's better here - thanks